include/mbgl/darwin/MGLTypes.h

```diff
@@ -26,6 +26,15 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingMode) {
     MGLUserTrackingModeFollowWithCourse,
 };
 
+/** The style layer name for a queried map feature. */
+extern NSString * const MGLFeatureLayerNameKey;
```

Comment:
Unlike `MGLUserTrackingMode`, these keys are only used by MGLMapView, so they can go in MGLMapView.h.